src/utils/storage/index.ts

```diff
@@ -17,15 +17,17 @@ enum StorageKeysEnum {
   ProfileData = 'profiledata',
   OtherProfileData = 'otherprofiledata',
   LastPromptNotification = 'lastPromptNotification',
-  LastSelectedMenu = 'lastSelectedMenu'
+  LastSelectedMenu = 'lastSelectedMenu',
+  BlockedStatus = 'blockedStatus',
+  BlockingStatus = 'blockingStatus'
 }
 
 interface IStorage {
-  get: () => string;
+  get: () => string | undefined;
   set: (value: string) => void;
   clear: () => void;
   setForKey: (key: string, value: string) => void;
-  getForKey: (key: string) => string;
+  getForKey: (key: string) => string | undefined;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the return type of `get` and `getForKey` methods from `string` to `string | undefined` is a significant one. This change implies that these methods can now return `undefined`, which might not be handled in all places where these methods are used. It's important to ensure that all usages of these methods are checked and updated to handle the `undefined` case if necessary.



<!-- This is an auto-generated comment by OSS CodeRabbit -->